docs-source/spring/content/platform/config/_index.md

```diff
@@ -1,32 +1,395 @@
-  application:
-    name: atael
-  cloud:
-    config:
-      label: latest
-      profile: dev
 ```
 
-This example fetches data where the application is `atael`, profile is `dev` and the label is `latest`.
+Configuration is stored in the `CONFIGSERVER` schema in the `PROPERTIES` table. Managing the data for the Spring Cloud Config server should be done using the CLI or via the REST API endpoints. If you prefer, you can also work directly with the `CONFIGSERVER.PROPERTIES` table in the database. How to access the database is documented here, ([Accessing the database](../../database/)).
 
-Managing the data for the Spring Cloud Config server should be done using the CLI. If you prefer, you can also work directly
-with the `CONFIGSERVER.PROPERTIES` table in the database.
+During setup of Oracle Backend for Spring Boot and Microservices the following data is loaded into `CONFIGSERVER.PROPERTIES`. This data can be deleted.
 
+```code
+| APPLICATION     | PROFILE        | LABEL    | PROP_KEY            | VALUE
+|-----------------|----------------|----------|---------------------|-----------------------------------|
+| atael           | dev            | latest   | test-property       | This is the test-property value   |  
+| atael           | dev            | latest   | test-property-2     | This is the test-property-2 value |
+| application-a   | production     | 12c      | db-name             | databasename-a-prod               |
+| application-a   | production     | 12c      | db-connection       | connectionstring-a-prod           |
+| application-a   | development    | 23cbeta  | db-dev-name         | databasename-a-dev                |
+| application-a   | development    | 23cbeta  | db-dev-connection   | connectionstring-a-dev            |
+| application-b   | production     | 19c      | db-name             | databasename-b-prod               |
+| application-b   | production     | 19c      | db-connection       | connectionstring-b-prod           |
+| application-b   | development    | 23cbeta  | db-dev-name         | databasename-b-dev                |
+| application-b   | development    | 23cbeta  | db-dev-connection   | connectionstring-b-dev            |
+| application-c   | secret         | 23.4     | json-db             | 23c-json-db                       |
+| application-c   | secret         | 23.4     | json-sdb-conn       | 23c-mongo-conn                    |
+| application-c   | secret         | 23.4     | txenventq           | 23c-conn-string                   |
+| application-c   | secret         | 23.4     | txeventq            | 23c-kafka-name                    |
+```
+
+## Config Server REST endpoints overview
+
+The following REST Endpoints are available to Config Server entries. The table lists which minimum required role that is needed to perform the operation. `N/A` in the table below means that endpoint doesn't require authentication to be accessed.
```

Comment:
Change to:

The following REST API endpoints are available to the Config Server entries. The table lists which minimum required role is needed to perform the operation. 'N/A' in the following table indicates that the endpoint does not require authentication to be accessed.

Reply:
Done